internal/kafka/internal/workers/kafka_mgrs/accepted_kafkas_mgr.go

```diff
@@ -30,13 +30,14 @@ type AcceptedKafkaManager struct {
 }
 
 // NewAcceptedKafkaManager creates a new kafka manager
```

Comment:
Looking at the existing comment, it looks general. 
I think it is something that can it is something that can be improved.
```suggestion
// NewAcceptedKafkaManager creates a new kafka manager to reconcile accepted kafkas
```
Same for this https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/f8e2a70b344f460abde5b36e1e4345a6e4f4f8fb/internal/kafka/internal/workers/kafka_mgrs/accepted_kafkas_mgr.go#L22

And I think it'll be done throughout the [kafka_mgrs](https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/f8e2a70b344f460abde5b36e1e4345a6e4f4f8fb/internal/kafka/internal/workers/kafka_mgrs/)
except for the kafkas_mgr.go file. This can be done in a separate PR.

What do you think?